Library/Formula/fb-infer.rb

```diff
@@ -0,0 +1,53 @@
+class FbInfer < Formula
+  desc "Static analyzer for Java, C and Objective-C"
+  homepage "http://fbinfer.com/"
+  url "https://github.com/facebook/infer/releases/download/v0.5.0/infer-osx-v0.5.0.tar.xz"
+  sha256 "6a8547ac0b75a5e2bbeccae2169e39f753a60adbcacb6c94599fd31343a71ce7"
+  head "https://github.com/facebook/infer.git"
+
+  option "with-clang-only", "Installs Infer with only Clang support."
+  option "with-java-only", "Installs Infer with only Java support."
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "opam" => :build
+
+  def install
+    target_language = "all"
+    if build.with?("clang-only") && build.with?("java-only")
+      odie "Options --with-clang-only and --with-java-only are mutually exclusive."
+    end
+
+    if build.with?("java-only")
+      target_language = "java"
+    elsif build.with?("clang-only")
+      target_language = "clang"
+    end
```

Comment:
This is purely cosmetic, but why have you decided to test for `java-only` first where in all other instances `clang-only` comes first. (Feel free to ignore this, just wanted to point it out since the formula needs a few more tweaks anyway.)